pyproject.toml

```diff
@@ -63,11 +63,15 @@ test = [
   'setuptools >= 67.8.0; python_version >= "3.12"',
 ]
 typing = [
+  "build[uv]",
   "importlib-metadata >= 5.1",
   "mypy ~= 1.5.0",
   "tomli",
   "typing-extensions >= 3.7.4.3",
 ]
+uv = [
+  "uv >= 0.1.6",
```

Comment:
Why not 0.1.15?

Reply:
Can do, v0.1.6 was the version which exposed `find_uv_bin`: https://github.com/astral-sh/uv/commit/469ccf826f9e58d452877fb0c2888d189423e040 (familiar name :)

Reply:
Unless we run the test suite with 0.1.6 I'd prefer to use latest the CI actually runs again... Otherwise, how do you know it works?

Reply:
I would've added it to our `constraints.txt` but I don't think we need to be backward-compatible with uv versions prior to our supporting it - I'll bump the version number.

Reply:
I've done that [here](https://github.com/layday/instawow/blob/896dfcab3a757a8e3849096d8cab03fcb0dafe14/noxfile.py#L81-L83). The only trouble is that it doesn't have a second-level dependency resolution option so you've got to pluck the package's direct dependencies out of the wheel metadata (or `pyproject.toml`).